echopype/echodata/echodata.py

```diff
@@ -191,7 +191,7 @@ def compute_range(
         Returns
         -------
         xr.DataArray
-            The range of the data in meters.
+            The echo range of the data in meters.
```

Comment:
Here, I think I'd would say: "The range (``echo_range``) of the data in meters."